### PR TITLE
Remove wizardtemplate.spec

### PR DESCRIPTION
**Pull request by dmarcoux**
We don't use this file. It's been untouched for 12 years...

It used to be part of a wizard workflow. :mage:

### CODE REVIEW

File: dist/obs-server.spec [hunk 1]
```diff
@@ -931,7 +931,6 @@ usermod -a -G docker obsservicerun
 %{__obs_api_prefix}/db/data
 %{__obs_api_prefix}/db/migrate
 %{__obs_api_prefix}/db/seeds.rb
-%{__obs_api_prefix}/files/wizardtemplate.spec
```
Comment (hennevogel) [inline dist/obs-server.spec:934]: Breaks the package build...

**Review by hennevogel — CHANGES_REQUESTED** [review]

File: dist/obs-server.spec (hunk 1 above)
Comment (dmarcoux) [reply dist/obs-server.spec:934]: The package build is failing due to those errors:
```
RPM build errors:
     File listed twice: /usr/lib/obs/server/BSConfig.pm
     Directory not found: /home/abuild/rpmbuild/BUILDROOT/obs-server-2.11~alpha.20200901T093837.fab91c9552-2.1.x86_64/srv/www/obs/api/files
```

Comment (hennevogel) [reply dist/obs-server.spec:934]: Yeah, you remove `files` directory but we explicitly package that somewhere...

Comment (dmarcoux) [reply dist/obs-server.spec:934]: Oh right, I forgot to remove that directory from the spec file.

**Comment by dmarcoux** [comment]
@hennevogel Could you have another look please?

**Review by hennevogel — APPROVED** [review]

**Pull request merged**